src/type/scalars.js

```diff
@@ -66,13 +66,20 @@ export const GraphQLInt = new GraphQLScalarType({
   serialize: serializeInt,
   parseValue: coerceInt,
   parseLiteral(ast) {
-    if (ast.kind === Kind.INT) {
-      const num = parseInt(ast.value, 10);
-      if (num <= MAX_INT && num >= MIN_INT) {
-        return num;
-      }
+    if (ast.kind !== Kind.INT) {
+      throw new TypeError(
+        `Int cannot represent non-integer value${
+          ast.value === undefined ? '.' : ': ' + inspect(ast.value)
```

Comment:
since this expression is at the end of the string, I think simple string concatenation with `+` would be more readable.

Reply:
Fixed

Reply:
Thanks